content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -766,18 +766,13 @@ You can see the name of the PVC bound to the PV using `kubectl describe persiste
 
 #### Phase transition timestamp
 
-{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+{{< feature-state for_k8s_version="v1.31" state="stable" >}}
```

Comment:
Improvement for 'feature-state' shortcode usage here, consider using the new `feature_gate_name` option, making the feature state more data-driven.

```suggestion
{{< feature-state feature_gate_name="PersistentVolumeLastPhaseTransitionTime" >}}
```

Reply:
Done.